src/views/v2/event-status/single/event-statuses.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Event Status Container.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/single/event-statuses.php
```

Comment:
Maybe name this `status` rather than `statuses`? I only see one status here.

Reply:
It is the gateway to multiple statuses. It uses the status id to get the template